e2e/packages/sync-test/data/waitForInitialSync.ts

```diff
@@ -3,4 +3,6 @@ import { Page, expect } from "@playwright/test";
 export async function waitForInitialSync(page: Page) {
   const syncStep = page.locator("#sync-step");
   await expect(syncStep).toHaveText("live");
+  const syncProgress = page.locator("#sync-percentage");
+  await expect(syncProgress).toHaveText("100");
```

Comment:
do we need to test for this?

I kinda wonder about keeping the "live" indicator (caught up) separate from the percentage synced, so you can e.g. hide your load screen (because "live") but potentially show how "up to date" you are (percentage)

Reply:
I think we can remove this test. `createStoreSync.test.ts` does it more directly